pkg/core/location/api.go

```diff
@@ -6,9 +6,11 @@ import (
 	"github.com/anexia-it/go-anxcloud/pkg/client"
 )
 
-// API contains methods for location listing.
+// API contains methods for getting and listing location.
```

Comment:
While `getting` clearly communicated the HTTP verb, I think `retrieving` sounds a bit better. Also, in this sentence,`locations` would be better than `location`.

Reply:
Done via cc1f6a6

Reply:
Only the first sentence though ;)